airstory.php

```diff
@@ -16,6 +16,9 @@
 
 namespace Airstory;
 
+// Declare the canonical plugin version.
+define( 'AIRSTORY_VERSION', '0.1.0' );
```

Comment:
I'm under the assumption this will be changed to 1.0.0 when it is released to the .org repo.